src/robusta/integrations/msteams/msteams_mark_down_fix_url.py

```diff
@@ -1,26 +1,23 @@
-
+import logging
```

Comment:
Why did you add it?

Reply:
Whoops, fixed it. (It was from an earlier version when I was debugging the issue)